docs/modules/ROOT/pages/ios_troubleshooting.adoc

```diff
@@ -0,0 +1,110 @@
+= Troubleshooting
+:toc: right
+:keywords: troubleshooting, logging, debugging, mitmproxy, charles for iOS, ownCloud, iOS, iPhone, iPad
+:description: This guide steps you through how to troubleshoot issues with ownCloud's iOS App for iPhone and iPad. Specifically, it shows how to configure logging, and troubleshoot using mitmproxy and Charles for iOS.
+:apache-logging-url: http://httpd.apache.org/docs/current/logs.html
+:charles-web-debugging-proxy-url: https://www.charlesproxy.com/documentation/ios/
+:create-screen-recording-url: https://support.apple.com/en-us/HT207935
+:mitmproxy-url: https://mitmproxy.org/
+:owncloud-logging-url: https://doc.owncloud.com/server/10.0/admin_manual/configuration/server/logging_configuration.html
+
+If you experience problems while using the iOS app, you can use this guide to help find the solution.
+
+== Logging
+
+=== Capturing App Debug Logs
+
+Effectively debugging software requires as much relevant information as possible.
+Log output can help with tracking down problems and, if you report a bug, log output can help to resolve an issue more quickly.
+To assist the ownCloud support personnel, please try to provide as many relevant logs as possible.
+You can do this by enabling and fully configuring the iOS app’s logging functionality via:
+
+* Enabling logging, _if it is not already enabled_.
+* Setting the log level to "_Debug_" or "_Info_".
+* Then, enabling:
+** Log HTTP requests and responses.
+** Standard error output.
+** Log file.
+
+Once these have been enabled:
+
+* Click btn:[Reset log file].
+* Perform the steps to reproduce the error
+* Go back to the Logging settings and click btn:[Share log file].
+
+.All iOS App logging settings enabled and set
+image:ios-app-settings-logging.png[ios-app-settings-logging]
```

Comment:
Screenshot outdated. It seems to be taken before the Log rotation feature

Reply:
Won't fix for now – screenshots need to be updated anyway